common/error.go

```diff
@@ -73,3 +84,19 @@ func (p ErrorPostProcessor) handleError(err error) error {
 		return err
 	}
 }
+
+// Errors coming from Oauth2 library are mapped here.
```

Comment:
Seems like this is the default, can you please mention it here in that case?

Reply:
I changed and added some more comments. Let me know if it is better.